tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
```

Comment:
OK I have added a mechanism for adding as many source map retrievers as we want:
https://github.com/meteor/meteor/blob/5f5ee2b015a89ddc20dbf860006498f706aaa01f/tools/source-map-retriever-stack.js

Not sure if this is overengineering, but it seemed like the most reasonable thing to do in a situation where the source map support needs to be globally installed.